template/scripts/swagger-codegen.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Please include a section in the README which describes how to use swagger codegen